app/src/main/java/eu/kanade/tachiyomi/ui/more/MoreController.kt

```diff
@@ -170,9 +170,11 @@ class MoreController :
     }
 
     private fun updateDownloadQueueSummary(preference: Preference) {
+        var pendingDownloadExists = downloadQueueSize != 0
+
         preference.summary = when {
-            downloadQueueSize == 0 -> null
-            !isDownloading -> resources?.getString(R.string.paused)
+            !pendingDownloadExists -> null
```

Comment:
The variable here is not really needed, could be reverted back to
```suggestion
           downloadQueueSize == 0 -> null
```